caluma/caluma_workflow/filters.py

```diff
@@ -79,6 +80,7 @@ class CaseFilterSet(MetaFilterSet):
     order_by = OrderingFilter(label="CaseOrdering", fields=("status",))
 
     document_form = CharFilter(field_name="document__form_id")
+    document_forms = GlobalIDMultipleChoiceFilter(field_name="document__form_id")
```

Comment:
The `GlobalIDMultipleChoiceFilter` is not the correct filter type: Forms have slugs, so the filter parameter should be string. The global id filters apply additional magic to parse the global ID - and in this case it also has the wrong type:

`documentForm: String` vs `documentForms: [ID]`

There's a `SlugMultipleChoiceFilter` that would be suitable for the given purpose, I suggest using that one instead

Reply:
Done